warehouse/views.py

```diff
@@ -0,0 +1,42 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pyramid.view import view_config
+
+
+@view_config(route_name='security', renderer='security.html')
+def security(request):
+    return {
+        'administrators': [
```

Comment:
@vladimir-v-diaz tells me the User model has name and email, but not keyid and fingeprint. I'm not sure whether we should add those fields to the model just to accommodate the administrators. What do you think, @domenkozar ?